tests/Unit/DependencyInjection/DbToolsExtensionTest.php

```diff
@@ -12,7 +12,7 @@
 
 class DbToolsExtensionTest extends TestCase
 {
-    private function getContainer(array $parameters = [], array $bundles = [])
+    private function getContainer(array $parameters = [], array $bundles = []): \Symfony\Component\DependencyInjection\ContainerBuilder
```

Comment:
We prefere to have a use statement at the top of the file and then only use the class name

```suggestion
use Symfony\Component\DependencyInjection\ContainerBuilder;

//...
    
    private function getContainer(array $parameters = [], array $bundles = []): ContainerBuilder
```

Reply:
This is part of other Pull Request with the rector rule "[import names](https://getrector.com/documentation/import-names)". I can apply here in this PR if you want, but probably changes others in code too that are present.

Reply:
I just sent https://github.com/makinacorpus/DbToolsBundle/pull/126 as you can see the result

Reply:
No, we don't want to apply this rules everywhere, you can just fix it in this PR.

Reply:
Done